Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractionTests_Press.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.Playables;
+using UnityEngine.TestTools;
+using System.Collections;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class InteractionTests_Press
+    {
+        // /// Test InteractionTest_Press_Far
```

Comment:
needed?

Reply:
Must have missed this, removed now. I'm going to look into writing proper tests for the recording feature.